packages/devextreme/testing/tests/DevExpress.ui.widgets/treeView.markup.tests.js

```diff
@@ -55,13 +55,17 @@ QUnit.module('aria accessibility', {
     }
 }, () => {
     QUnit.test('aria role', function(assert) {
-        assert.equal(this.$element.attr('role'), 'tree', 'role is correct');
+        assert.strictEqual(this.$element.attr('role'), 'tree', 'role is correct');
+
+        this.instance.option({ items: [] });
+
+        assert.strictEqual(this.$element.attr('role'), undefined, 'role is not set');
     });
 
-    QUnit.test('scrollable should have role treeitem attribute', function(assert) {
-        const $scrollable = this.$element.find('.' + SCROLLABLE_CLASS);
+    QUnit.test('scrollable should not have role attribute', function(assert) {
```

Comment:
I believe this test is not necessary now.
Seems strange to me that we test that element does not have an attribute.

Reply:
I think this test could be useful if we want to set a role attribute in the future. Right now we know it's wrong. But we have added Axe tests, so I agree with you that we can get rid of this test.

Reply:
[3fe0ac7ff44946354ba94acc5981fa062abc353d]